product_docs/docs/biganimal/release/reference/cli/index.mdx

```diff
@@ -82,19 +82,61 @@ You can create multiple credentials for different BigAnimal accounts and then se
 ```shell
 biganimal credential show
 __OUTPUT__
-┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ Credentials                                                             ┃
-┣━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━┫
-┃ name     ┃ address                     ┃ port ┃  context credential     ┃
-┃━━━━━━━━━━┃━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┃━━━━━━┃━━━━━━━━━━━━━━━━━━━━━━━━━┃
-┃ ba-user1 ┃ portal.biganimal.com        ┃ 443  ┃ x                       ┃
-┃ ba-user2 ┃ portal.biganimal.com        ┃ 443  ┃                         ┃
-┗━━━━━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━┻━━━━━━━━━━━━━━━━━━━━━━━━━┛
+┌────────────────────────────────────────────────────────────────────────────────────┐
+│ Credentials                                                                        │
+├──────────┬──────────────────────┬──────────────────────┬──────┬────────────────────┤
+│ name     │ Organization         │ address              │ port │ context credential │
+├──────────┼──────────────────────┼──────────────────────┼──────┼────────────────────┤
+│ ba-user1 │ My_organization      │ portal.biganimal.com │ 443  │ x                  │
+│ ba-user2 │ My_organization      │ portal.biganimal.com │ 443  │                    │
+└──────────┴──────────────────────┴──────────────────────┴──────┴────────────────────┘
+
 The credential ba-user1 has been set up
 ```
 
 ```shell
 biganimal config set context_credential ba-user1
+```
+## Creating credentials for an alternative organization
+
+The BigAnimal CLI supports the capability to create credentials for an alternative organization to which you have been invited or are already a member. You can invite users that have an EDB account to join an organization by selecting the **Invite New User** option on the Users page in the BigAnimal portal.
+
+To create credentials for an alternative organization, you can use either interactive mode or specify the settings with flags on the command line:
+
+- Select the organization from the list in interactive mode:
+
+  ```shell
+  biganimal credential create 
+  __OUTPUT__
+  ? Credential Name: ba-user2
+  ? Organization:  [Use arrows to move, type to filter]
+    My_Organization
+  > org_aNR9SNuNcUN6vGSn
```

Comment:
Hello, this section is supposed to always output the organization name instead of the organization ID. 

Where did you get the output from?

Reply:
the `org_aNR9SNuNcUN6vGSn` stuff is always the organization ID and supposes to be not shown here. (unless there is a bug :sob:)

Reply:
Hello, @xugy99! Thank you for the feedback!
That is my mistake. I had an organization name showing as `Fran Coughlin's organization` and wanted to replace it in the example with a different organization. I used credential show, and `org_aNR9SNuNcUN6vGSn` appeared in the output under `Organization`, so I mistakenly assumed it was a valid organization name. I believe I have corrected it in this latest commit?
Thank you!
Fran